packages/bpk-component-accordion/src/BpkAccordionItem.module.scss

```diff
@@ -24,7 +24,13 @@ $bpk-spacing-v2: true;
   &__title {
     height: auto;
 
-    @include bpk-border-bottom-sm($bpk-line-day);
+    &--expanded {
+      @include bpk-border-bottom-sm($bpk-line-day);
+
+      &-day {
+        @include bpk-border-bottom-sm($bpk-line-on-dark-night);
```

Comment:
Night tokens are not to be used outside of dark mode, can you check the designs of what should be used here and if it's a night token we can raise to design